ServerCore/Pages/PuzzleApiController.cs

```diff
@@ -38,5 +43,27 @@ public async Task<ActionResult<bool>> GetValidateTeamPasswordAsync(string teamPa
                           where team.Password == teamPassword
                           select team).AnyAsync();
         }
+
+        [HttpPost]
+        [Authorize(Policy = "PlayerCanSeePuzzle")]
+        [Route("api/puzzleapi/submitanswer/{eventId}/{puzzleId}")]
+        public async Task<SubmissionResponse> PostSubmitAnswerAsync([FromBody] AnswerSubmission submission, [FromRoute] string eventId, [FromRoute] int puzzleId)
+        {
+            Event currentEvent = await EventHelper.GetEventFromEventId(context, eventId);
+
+            PuzzleUser user = await PuzzleUser.GetPuzzleUserForCurrentUser(context, User, userManager);
```

Comment:
Is this possible with auth enabled?

Reply:
In what way?

Reply:
Comment is off by a line. The question is whether user would ever be null given that auth is enabled